consoles/sshVirtsh.pm

```diff
@@ -325,111 +325,169 @@ sub add_interface {
     return;
 }
 
+sub to_byte {
+    my $sp              = shift;
+    my $size_multiplier = {
+        b => 1,
+        k => 1024, K => 1024,
+        M => 1024 * 1024,
+        G => 1024 * 1024 * 1024,
+        T => 1024 * 1024 * 1024 * 1024,
+    };
```

Comment:
https://metacpan.org/pod/Number::Format might be more helpful here. There is also https://metacpan.org/pod/Number::Bytes::Human which is currently not in openSUSE:Factory but we could submit that

Reply:
> https://metacpan.org/pod/Number::Format might be more helpful here. There is also https://metacpan.org/pod/Number::Bytes::Human which is currently not in openSUSE:Factory but we could submit that

`Number::Format` seems to work nicely, thanks!